packages/@okta/vuepress-site/docs/guides/federated-claims/main/index.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Federated claims with entitlements
+excerpt: Learn how to implement federated claims with entitlements for an app
+layout: Guides
+---
+
+This guide describes how to configure your app to pass entitlements in your tokens using federated claims.
+
+---
+
+#### Learning outcomes
+
+* Understand the purpose of federated claims with entitlements.
+* Understand how to configure federated claims for a SAML and an OpenID Connect (OIDC) app.
+
+#### What you need
+
+* [Okta Developer Edition organization](https://developer.okta.com/signup)
+* The Identity Governance feature enabled for your org. Contact [Okta Support](https://support.okta.com) to enable this feature.
+* The Federated Claim Generation Layer feature enabled for your org. To enable this feature, go to **Settings** > **Features**, locate the Federated Claim Generation Layer feature, and enable.
+* An existing SAML or OIDC app
+* A test user assigned to the SAML or OIDC app
+
+---
+
+## Overview
+
+Federated claims create a more consistent experience for the configuration of claims across protocols. They unify the inconsistencies in the persistence and generation of token claims by adding support for a claim type that all apps can consume.
```

Comment:
"unify the inconsistencies in the persistence and generation of token claims by adding support for" is hard for me to follow. Lots of prep phrases to unpack. 

Maybe also the repeated phonetics of "-istence." I think this sentence could use a simplification.